html-files/pages/Page14.html

```diff
@@ -0,0 +1,218 @@
+                                class="u-expanded-width u-image u-image-contain u-image-default u-image-5 img-hover-effect"
+                                data-image-width="601" data-image-height="390">
+                            <h4 class="u-text u-text-default u-text-palette-1-base u-text-10">Iced Coffee</h4>
+                            <p class="u-align-center u-text u-text-palette-1-dark-2 u-text-11">Iced coffee is a coffee beverage served cold. It is prepared by brewing coffee normally and then serving it over ice and adding it in cold milk or by brewing the coffee cold.
+                            </p>
+                            <a href="https://youtu.be/Bh4nXflcVw0"
+                                class="u-active-palette-1-light-1 u-border-2 u-border-active-white u-border-hover-white u-border-white u-btn u-btn-round u-button-style u-hover-palette-1-light-1 u-palette-1-base u-radius-50 u-text-active-white u-text-hover-white u-btn-6"
+                                target="_blank"> Take me there</a>
+                        </div>
+                    </div>
+                    <div
+                        class="u-align-center u-container-style u-list-item u-radius-20 u-repeater-item u-shape-round u-video-cover u-white u-list-item-3">
+                        <div class="u-container-layout u-similar-container u-valign-top u-container-layout-5">
+                            <img src="/images/Page 14/soyaChaap.png" alt="soya chaap preview"
+                                class="u-expanded-width u-image u-image-contain u-image-default u-image-5 img-hover-effect"
+                                data-image-width="601" data-image-height="390">
+                            <h4 class="u-text u-text-default u-text-palette-1-base u-text-10">Malai Soya Chaap</h4>
+                            <p class="u-align-center u-text u-text-palette-1-dark-2 u-text-11">Malai Soya chaap curry is rich North Indian curry made with soya chaap. It is a vegetarian recipe cooked in the white gravy made with cream, curd, cashew paste and other spices.</p>
+                            <a href="https://youtu.be/5zkpY2KrV7g"
+                                class="u-active-palette-1-light-1 u-border-2 u-border-active-white u-border-hover-white u-border-white u-btn u-btn-round u-button-style u-hover-palette-1-light-1 u-palette-1-base u-radius-50 u-text-active-white u-text-hover-white u-btn-6"
+                                target="_blank"> Take me there</a>
+                        </div>
+                    </div>
+                    <div
+                        class="u-align-center u-container-style u-list-item u-radius-20 u-repeater-item u-shape-round u-video-cover u-white u-list-item-3">
+                        <div class="u-container-layout u-similar-container u-valign-top u-container-layout-5">
+                            <img src="/images/Page 14/Uttapam.png" alt="uttapam preview"
```

Comment:
- The Capitalization of the word ```U``` in ```uttapam.png``` is causing image loading issues. Fix it, and send in the updated fix, in the same PR (do not close this one).

![image](https://github.com/TheGuyDangerous/FoodCoders/assets/78858516/44bdc2d3-e441-4c53-bf57-00ba611b1062)